CMakePresets.json

```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "https://cmake.org/cmake/help/latest/_downloads/3e2d73bff478d88a7de0de736ba5e361/schema.json",
+	"version": 8,
+	"cmakeMinimumRequired": {
+		"major": 3,
+		"minor": 28
```

Comment:
Because you defined it here as 3.28 being the minimum CMake version of the project, can you update all the places where `cmake_minimum_required(VERSION 3.15)` is present (once in the main CMake and a few times in the libraries CMake)

Reply:
> Because you defined it here as 3.28 being the minimum CMake version of the project

This is the minimum version of CMake required to use this preset (since schema version 8 was [only added in 3.28](https://cmake.org/cmake/help/latest/manual/cmake-presets.7.html#format)), not build the project itself